import-export-cli/cmd/mg.go

```diff
@@ -0,0 +1,46 @@
+/*
+*  Copyright (c) WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+*
+*  WSO2 Inc. licenses this file to you under the Apache License,
+*  Version 2.0 (the "License"); you may not use this file except
+*  in compliance with the License.
+*  You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied.  See the License for the
+* specific language governing permissions and limitations
+* under the License.
+ */
+
+package cmd
```

Comment:
Shall we move Microgateway related code segements to a seperate package? (Eg: mg)
You can refer [1] where MI code segments are located. Also, currently the k8s related code segments are being moved as well to a seperate package named "k8s".

[1] https://github.com/wso2/product-apim-tooling/tree/master/import-export-cli/cmd/mi